ebook/2_html2epub.cmd

```diff
@@ -0,0 +1,12 @@
+@echo off
```

Comment:
Please can we not have a `.cmd` script that does the same thing as a shell script? I think it's not unreasonable these days on Windows to run a shell script; there are plenty of ways to do so.

Reply:
ok, you are the boss ;-)